packages/edit-widgets/src/store/actions.js

```diff
@@ -15,12 +15,41 @@ import { dispatch, select, getWidgetToClientIdMapping } from './controls';
 import { transformBlockToWidget } from './transformers';
 import {
 	buildWidgetAreaPostId,
+	buildWidgetAreasPostId,
 	buildWidgetAreasQuery,
+	createStubPost,
 	KIND,
 	POST_TYPE,
 	WIDGET_AREA_ENTITY_TYPE,
 } from './utils';
 
+/**
+ * Initializes the stub post before rendering the widgets editor.
+ * Required in order to ensure the data layer won't try to resolve the stub post.
+ *
+ * @access private
+ */
+export function* initializeState() {
+	yield persistStubPost( buildWidgetAreasPostId(), [] );
+}
+
+export const persistStubPost = function* ( id, blocks ) {
+	const stubPost = createStubPost( id, blocks );
+	const args = [ KIND, POST_TYPE, id ];
+	yield dispatch( 'core', 'startResolution', 'getEntityRecord', args );
```

Comment:
I know from experience that this is the magic that prevents `core-data` from trying to resolve the entity, but I imagine most would not pick that up. Let's add a doc comment.